packages/salesforcedx-vscode-core/src/util/inputUtils.ts

```diff
@@ -0,0 +1,8 @@
+import * as vscode from 'vscode';
```

Comment:
Please add the copyright here.  Just copy-paste from one of the other files, and make sure the year is 2022 since this is a new file.